internal/server/data/passwordresettoken.go

```diff
@@ -4,63 +4,86 @@ import (
 	"errors"
 	"time"
 
-	"gorm.io/gorm"
-
 	"github.com/infrahq/infra/internal"
 	"github.com/infrahq/infra/internal/generate"
 	"github.com/infrahq/infra/internal/logging"
 	"github.com/infrahq/infra/internal/server/models"
 	"github.com/infrahq/infra/uid"
 )
 
-func CreatePasswordResetToken(db GormTxn, user *models.Identity, ttl time.Duration) (*models.PasswordResetToken, error) {
+type passwordResetToken struct {
+	ID uid.ID
+	models.OrganizationMember
+
+	Token      string
+	IdentityID uid.ID
+	ExpiresAt  time.Time
+}
+
+func (passwordResetToken) Table() string {
+	return "password_reset_tokens"
+}
+
+func (p passwordResetToken) Columns() []string {
+	return []string{"expires_at", "id", "identity_id", "organization_id", "token"}
+}
+
+func (p passwordResetToken) Values() []any {
+	return []any{p.ExpiresAt, p.ID, p.IdentityID, p.OrganizationID, p.Token}
+}
+
+func (p *passwordResetToken) ScanFields() []any {
+	return []any{&p.ExpiresAt, &p.ID, &p.IdentityID, &p.OrganizationID, &p.Token}
+}
+
+func (p *passwordResetToken) OnInsert() error {
+	return nil
+}
+
+func CreatePasswordResetToken(db GormTxn, user *models.Identity, ttl time.Duration) (string, error) {
```

Comment:
TODO: check user.ID and ttl are not zero values.

The old model had struct tags for this, but they were not used by anything since we've removed go-playground/validator